airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2622 @@
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/workspaces_partition_router"
+
+  teams_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/teams_stream"
+        parent_key: "gid"
+        partition_field: "team_gid"
+
+  team_memberships_stream:
+    $ref: "#/definitions/base_stream"
+    name: "team_memberships"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/team_memberships_schema"
+    $parameters:
+      path: "/teams/{{ stream_slice['team_gid'] }}/team_memberships"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/teams_partition_router"
+
+  attachments_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "attachments_compact"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/attachments_compact_schema"
+    $parameters:
+      path: "/attachments"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      requester:
+        $ref: "#/definitions/requester"
+        use_cache: true
+        request_parameters:
+          parent: "{{ stream_slice.parent_gid }}"
+      partition_router:
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/projects_stream"
+              parent_key: "gid"
+              partition_field: "parent_gid"
+            - stream: "#/definitions/tasks_stream"
+              parent_key: "gid"
+              partition_field: "parent_gid"
+
+  attachments_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/attachments_compact_stream"
+        parent_key: "gid"
+        partition_field: "attachments_gid"
+
+  attachments_stream:
+    $ref: "#/definitions/base_stream"
+    name: "attachments"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/attachments_schema"
+    $parameters:
+      path: "/attachments/{{ stream_slice['attachments_gid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/attachments_compact_partition_router"
+
+  portfolios_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios_compact"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_compact_schema"
+    $parameters:
+      path: "/portfolios"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
```

Comment:
Missing request parameter: `opt_fields`. Please add it according to the original implementation.